Library/Homebrew/os/mac/cmd/config.rb

```diff
@@ -0,0 +1,196 @@
+#:  * `config`:
+#:    Show Homebrew and system configuration useful for debugging. If you file
+#:    a bug report, you will likely be asked for this information if you do not
+#:    provide it.
```

Comment:
I guess it's not the time yet to comment on individual parts of the code, but I can't help but notice that this work in progress is still a bit messy. Namely:

- This file `os/mac/cmd/config.rb` clearly shouldn't be here.
- There's at least one instance where two `private`s immediately follow each other.
- Maybe more? It's hard to spot this stuff when things are moved around and refactored in the same commit, causing Git to lose the connection between files (at least when viewed on GitHub).

Sorry for this pedantry …